docs/wallets/wallets.mdx

```diff
@@ -9,68 +10,199 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
 
 <img className="docsheader" src={useBaseUrl('img/wallets/header.png')} />
 
-# Helium Wallets
+## Helium Is Moving To Solana
 
-In blockchain projects, various wallet applications exist to help users manage their own private
-keys, enabling them to sign transactions and transfer assets.
+All Hotspots on the Helium Network with an animal name will automatically become a
```

Comment:
this feels like a very abrupt introduction to "Helium is moving to Solana". Can there be a little more preamble?

Reply:
preamble added, may need some more work. but also didn't want to deviate too far off topic from Wallet stuff to general L1 migration stuff